src/javascript/app/pages/trade/tick_trade.js

```diff
@@ -185,20 +208,20 @@ const TickDisplay = (() => {
                 // be 1.01
                 final_barrier = Number(`${Math.round(`${final_barrier}e${display_decimals}`)}e-${display_decimals}`);
                 barrier_quote = final_barrier;
-            } else if (abs_barrier) {
-                barrier_quote = parseFloat(abs_barrier);
+            } else if (contract && contract.barrier) {
+                barrier_quote = parseFloat(contract.barrier);
             }
 
             chart.yAxis[0].addPlotLine({
                 id    : 'tick-barrier',
                 value : barrier_quote,
-                label : { text: `Barrier (${addComma(barrier_quote)})`, align: 'center' },
+                label : { text: localize('Barrier ([_1])', [addComma(barrier_quote)]), align: 'center' },
```

Comment:
could we change this to `${localize('Barrier') (${addComma(barrier_quote)})` as well? similar to line 241